content/it/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
 
-Consulta le guide in [Scelta della soluzione giusta](/docs/setup/pick-right-solution/) per esempio su come pianificare, 
+Consulta le guide in [Setup](/docs/setup) per esempio su come pianificare, 
 impostare e configurare i cluster di Kubernetes. Le soluzioni elencate in questo articolo sono chiamate *distros*.
```

Comment:
`distros` -> `distribuzioni`

"Distros" is not commonly used to refer to a distribution. We call them `distribuzioni` (as per Linux distros). As we are changing this sentence I think it's worth to fix this and turn it into `*distribuzioni*`

Reply:
@jnardiello  done

Reply:
Thanks for the changes.